membership-common/src/main/scala/com/gu/memsub/ProductFamily.scala

```diff
@@ -16,6 +16,7 @@ object Product {
 
   sealed trait Paper extends ContentSubscription
   sealed trait Weekly extends Paper
+  sealed trait Newspaper extends Paper
```

Comment:
I've slotted a new Newspaper trait in so that I can pattern match on any paper but not pick up GW by accident.